Adding resource evaluation filters as a terraform resource (#3004)
DataDog/terraform-provider-datadog: docs/resources/resource_evaluation_filter.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "datadog_resource_evaluation_filter Resource - terraform-provider-datadog"
subcategory: ""
description: |-
  Manage a single resource evaluation filter.
---

# datadog_resource_evaluation_filter (Resource)

> Manage a single resource evaluation filter.

Reviewer: What does a resource evaluation filter do for a user? Let's explain that.

Reviewer: Hopefully we can just copy/paste from the existing Datadog docs.

Reviewer: Ok, I'm good with being consistent.

## Example Usage

```terraform
# Manage Datadog resource evaluation filters
resource "datadog_resource_evaluation_filter" "basic_filter" {
  tags           = ["tag1:val1"]
  cloud_provider = "aws"
  id             = "000000000000"
}
```

<!-- schema generated by tfplugindocs -->
## Schema

### Required

> - `cloud_provider` (String) The cloud provider of the resource that will be target of the filters.

Reviewer 2: ```suggestion
- `cloud_provider` (String) The cloud provider of the filter's targeted resource.
```

> - `id` (String) The ID of the resource that will be the target of the filters. Different cloud providers target different resource ids:

Reviewer 2: ```suggestion
- `id` (String) The ID of the of the filter's targeted resource. Different cloud providers target different resource IDs:
```

  - `aws`: account id 
  - `gcp`: project id
  - `azure`: subscription id

> - `tags` (Set of String) Set of tags to filter the misconfiguration detections on. Each entry should follow the format: "key:":"value".

Reviewer 2: ```suggestion
- `tags` (Set of String) Set of tags to filter misconfiguration detections. Each entry should follow the format: "key:":"value".
```

## Import

Import is supported using the following syntax:

```shell

> terraform import datadog_resource_evaluation_filer.test_filter aws:00000000000000

Reviewer 2: ```suggestion
terraform import datadog_resource_evaluation_filter.test_filter aws:00000000000000
```
